distribution/docker/README.md

```diff
@@ -92,6 +92,33 @@ images, and combining them with a Docker manifest. The Elasticsearch Delivery
 team aren't responsible for this - rather, it happens during our unified release
 process.
 
+To build multi-architecture images on `x86_64` hosts using Docker[^1], you'll
+need [buildx](https://docs.docker.com/build/buildx/install/) and ensure that it
+supports both `linux/amd64` **and** `linux/arm64` targets.
+
+You can verify the supported targets using `docker buildx ls`. For example, the
+following output indicates that support for `linux/arm64` is missing:
+
+```shell
+$ docker buildx ls
+NAME/NODE DRIVER/ENDPOINT STATUS  BUILDKIT PLATFORMS
+default * docker
+default default         running 20.10.21 linux/amd64, linux/386
```

Comment:
I think maybe something has happened to the column alignment here? Can you check please?

Reply:
Good catch. Fixed in 47bf866e6fd7fe0ec69b26738edc8251b155e4de